misc_scripts/update_workflow_config.py

```diff
@@ -59,7 +59,9 @@ def search_file(search_key, old_verstring, new_verstring, filename):
             if search_key in line:
                 templine = line.replace(old_verstring, new_verstring)
                 if new_verstring not in templine:
-                    raise RuntimeError(f'Old version "{old_verstring}" not in file')
+                    raise RuntimeError(
```

Comment:
While you're here, better make the error message as precise as you can. In particular, you want to say it's not in ```line``` in file ```filename```. Saying it is not in the file at all is overstating things.

Reply:
Quick experiment:
```
                if new_verstring not in templine:
                    raise RuntimeError(
                        f'Old version "{old_verstring}" not in "{line} in file "{filename}"'
                    )
```

Resulting output:
```
RuntimeError: Old version "1.79.0" not in "            toolchain: 1.72.0  # CURRENT DEVELOPMENT RUST TOOLCHAIN
 in file "../../stratisd/.github/workflows/fedora.yml"
```

Reply:
OK; amended the original commit to add this change.